README.md

```diff
@@ -1,3 +1,229 @@
 # LavaQueue
 
-yes, no docs lol.
\ No newline at end of file
+> [!IMPORTANT]
+> This plugin is compatible with Lavalink version four and higher.
+
+A plugin that adds a queue system for [Lavalink](https://github.com/lavalink-devs/Lavalink).
```

Comment:
A simple queue plugin for [LavaLink](https://github.com/lavalink-devs/Lavalink) with a REST API.